searchcore/src/vespa/searchcore/proton/reference/document_db_reference.cpp

```diff
@@ -5,11 +5,13 @@
 #include "gid_to_lid_change_registrator.h"
 #include <vespa/searchcore/proton/documentmetastore/documentmetastore.h>
 #include <vespa/searchlib/attribute/attributeguard.h>
-#include <vespa/searchlib/attribute/iattributemanager.h>
+#include <vespa/searchlib/attribute/imported_attribute_vector.h>
+#include <vespa/searchcore/proton/attribute/attributemanager.h>
+#include <vespa/searchcore/proton/attribute/imported_attributes_repo.h>
 
 namespace proton {
 
-DocumentDBReference::DocumentDBReference(std::shared_ptr<search::IAttributeManager> attrMgr,
+DocumentDBReference::DocumentDBReference(std::shared_ptr<AttributeManager> attrMgr,
```

Comment:
I suggest we use proton::IAttributeManager instead of proton::AttributeManager. The interface has the getImportedAttributes() function we need.